network-api/networkapi/wagtailpages/pagemodels/buyersguide/article_page.py

```diff
@@ -79,17 +78,8 @@ class BuyersGuideArticlePage(
 
     def get_context(self, request: http.HttpRequest, *args, **kwargs) -> dict:
         context = super().get_context(request, *args, **kwargs)
-        context['home_page'] = self.get_buyersguide_homepage()
         return context
 
-    def get_buyersguide_homepage(self):
```

Comment:
Removing the logic that was used in order to pass the home page into the context